guides/source/configuring.md

```diff
@@ -1671,108 +2183,209 @@ Because `Rails::Application` inherits from `Rails::Railtie` (indirectly), you ca
 
 Below is a comprehensive list of all the initializers found in Rails in the order that they are defined (and therefore run in, unless otherwise stated).
```

Comment:
I'm hesitant to convert this section's list items because it is intended to be an ordered list.  I feel like converting the items to subheadings somewhat diminishes that context.  What do you think?

Reply:
I've left it out for now.